README.md

```diff
@@ -133,15 +133,19 @@ TODO
 Get the most recent packaged binaries (`.deb`, `.rpm`, `.apk`, `pkg.tar.zst `, `.zip` or `.tar.gz` format) by visiting the [release page](https://github.com/tofuutils/tenv/releases). After downloading, unzip the folder and seamlessly integrate it into your system's `PATH`.
 
 <a id="docker-installation"></a>
+
 #### Docker Installation
-You can use dockerized version of tenv via the following commands:
 
-```shell
-TODO
+You can use dockerized version of tenv via the following command:
+
+```sh
+docker run -v ${HOME}/.tenv:/.tenv -u $(id -u) ghcr.io/tofuutils/tenv:latest -r /.tenv help
```

Comment:
I'm not quite sure why running this locally is actually useful -- it does download packages and change symlinks, however if you don't install the base OS package, the terraform, tofu and terragrunt commands won't exist on the system to pick up the files in ~/.tenv.

The main usecase I see would be using it from CI pipelines, where the image fortunately just works.

Reply:
Agree. I think that we can add it to documentation in the further commits. WDYT?

Reply:
I think thats a fine idea, I added a documentation suggestion in another comment, and then I suggest changing the commandline to something simpler to avoid confusion.

```suggestion
docker run --rm --it ghcr.io/tofuutils/tenv:latest help
```